src/apps/frontend/components/header/user-profile-snippet.component.tsx

```diff
@@ -59,7 +59,7 @@ const UserProfileSnippet: React.FC<DropdownUserProps> = ({
       >
         <span className="hidden text-right lg:block">
           <span className="block text-sm font-medium text-black dark:text-white">
-            {account.displayName()}
+            {account?.displayName()}
```

Comment:
Same thing - `account` should not be null or undefined when `user-profile` component is rendered